marklogic-data-hub/src/test/java/com/marklogic/hub/EndToEndFlowTests.java

```diff
@@ -425,16 +443,16 @@ public static void teardown() {
             }));
         }));
 
-        allCombos(((codeFormat, dataFormat, flowType) -> {
+        allCombos(((codeFormat, dataFormat, flowType, useEs) -> {
             String prefix = "validation-content-errors";
-            String flowName = getFlowName(prefix, codeFormat, dataFormat, flowType);
+            String flowName = getFlowName(prefix, codeFormat, dataFormat, flowType, useEs);
             tests.add(DynamicTest.dynamicTest(flowName, () -> {
                 // clear out the previous flows from above
                 FileUtils.deleteDirectory(entityDir.resolve("input").toFile());
                 FileUtils.deleteDirectory(entityDir.resolve("harmonize").toFile());
 
-                createFlow(prefix, codeFormat, dataFormat, flowType, (codeFormat1, dataFormat1, flowType1, srcDir, flowDir) -> {
-                    copyFile(srcDir + "content-syntax-error." + codeFormat.toString(), flowDir.resolve("content." + codeFormat.toString()));
+                createFlow(prefix, codeFormat, dataFormat, flowType, useEs, (codeFormat1, dataFormat1, flowType1, srcDir, flowDir, useEs1) -> {
```

Comment:
I see that `codeFormat1` isn't a new variable, but it is pretty opaque.

Reply:
suggestions?

Reply:
I've got nothing. leave it